From 8fb9b1be9f04a2297349061ab444b2de736e3e3b Mon Sep 17 00:00:00 2001
From: Lars Francke <lars.francke@stackable.tech>
Date: Tue, 9 Apr 2024 17:29:07 +0200
Subject: HIVE-26905: exclude pentaho-aggdesigner-algorithm

Backport HIVE-25173 to 3.2.0: Exclude pentaho-aggdesigner-algorithm from upgrade-acid build.
---
 upgrade-acid/pom.xml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/upgrade-acid/pom.xml b/upgrade-acid/pom.xml
index f95117e07d..b25b332b34 100644
--- a/upgrade-acid/pom.xml
+++ b/upgrade-acid/pom.xml
@@ -80,6 +80,12 @@
             <artifactId>hive-exec</artifactId>
             <version>2.3.3</version>
             <scope>provided</scope>
+            <exclusions>
+              <exclusion>
+                <groupId>org.pentaho</groupId>
+                <artifactId>pentaho-aggdesigner-algorithm</artifactId>
+              </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.apache.hadoop</groupId>
